### PR TITLE
[16.0][MIG] helpdesk_mgmt_sla

### PR DESCRIPTION
**Pull request by AntoniRomera**

### CODE REVIEW

File: helpdesk_mgmt_sla/views/helpdesk_ticket_team_views.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
         <field name="arch" type="xml">
             <xpath expr="//group[@name='right']" position="inside">
                 <field name="use_sla" />
-                <field name="resource_calendar_id" />
+                <field name="resource_calendar_id" groups="base.group_multi_company" />
```
Comment (pedrobaeza) [inline helpdesk_mgmt_sla/views/helpdesk_ticket_team_views.xml:10]: This is not correct. Add instead the field `company_id` without group as invisible.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Please also squash translation commits from the same translator and add the pre-commit stuff in a separate commit.

**Comment by andreampiovesana** [comment]
@hgarridoco can you fix?

**Comment by CarFla** [comment]
Hi.
this mean that this module will not be ported to odoo 16.0??

**Pull request closed without merging**